cmd/kubelet/app/server.go

```diff
@@ -1180,7 +1180,7 @@ func startKubelet(k kubelet.Bootstrap, podCfg *config.PodConfig, kubeCfg *kubele
 		go k.ListenAndServe(kubeCfg, kubeDeps.TLSOptions, kubeDeps.Auth, kubeDeps.TracerProvider)
 	}
 	if kubeCfg.ReadOnlyPort > 0 {
-		go k.ListenAndServeReadOnly(netutils.ParseIPSloppy(kubeCfg.Address), uint(kubeCfg.ReadOnlyPort))
+		go k.ListenAndServeReadOnly(netutils.ParseIPSloppy(kubeCfg.ReadOnlyBindAddress), uint(kubeCfg.ReadOnlyPort))
```

Comment:
Looks like the other usage of `ParseIPSloppy` above also trims space around the address, should we do that here too?

hm, TIL, interesting: https://github.com/kubernetes/kubernetes/issues/100895

Reply:
That instance is in the context of parsing a comma separated list of addresses. I don't think it is needed here.